tests/performance_tests/performance_test_basic.py

```diff
@@ -218,9 +230,27 @@ def prepArgs(self) -> dict:
         args.update(asdict(self.testHelperConfig))
         args.update(asdict(self.clusterConfig))
         args.update({key:val for key, val in inspect.getmembers(self) if key in set(['targetTps', 'testTrxGenDurationSec', 'tpsLimitPerGenerator',
-                                                                                     'expectedTransactionsSent', 'saveJsonReport', 'numAddlBlocksToPrune', 'quiet'])})
+                                                                                     'expectedTransactionsSent', 'delReport', 'numAddlBlocksToPrune', 'quiet', 'delPerfLogs'])})
         return args
 
+    def captureLowLevelArtifacts(self):
+        try:
+            shutil.move(f"var", f"{self.varLogsDirPath}")
+        except Exception as e:
+            print(f'Exception caught: {type(e)}: {e}')
```

Comment:
Rather than "Exception caught", say what didn't happen as expected.  f"Failed to move 'var' to {self.varLogsDirPath}"

Reply:
Addressed: [abc5c41](https://github.com/AntelopeIO/leap/pull/393/commits/abc5c4150d3fc732a8d6c51bfe555911d89d270b)